solutions/swb-app/src/dataSetRoutes.ts

```diff
@@ -69,6 +69,7 @@ export function setUpDSRoutes(router: Router, dataSetService: DataSetPlugin): vo
       await dataSetService.addDataSetExternalEndpoint({
         dataSetId: req.params.id,
         externalEndpointName: validatedRequest.externalEndpointName,
+        groupId: 'TODO',
```

Comment:
Still need this? Maybe we can add `groupId` to the parser and pass it in?

Reply:
I wasnt sure how SWB was going to implement this. From what I gather `groupId` is going to be the AuthZ group created for the project trying to access the dataset. I can add it to the request parser and handle it that way, or if theres a better way to do it, let me know.

Reply:
We are going to get the `groupId` from the request body